src/main/java/com/dreammaster/TwilightForest/TF_Loot_Chests.java

```diff
@@ -77,8 +77,8 @@ public static void init() {
         TFTreasure.hill3.uncommon.add(GT_OreDictUnificator.get(OrePrefixes.ingot, Materials.WroughtIron, 4L), 12);
         TFTreasure.hill3.uncommon.add(GT_OreDictUnificator.get(OrePrefixes.ingot, Materials.Steel, 4L), 12);
         TFTreasure.hill3.rare.add(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Thaumium, 1L), 12);
-        TFTreasure.hill3.rare.add(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminium, 2L), 12);
-        TFTreasure.hill3.rare.add(GT_OreDictUnificator.get(OrePrefixes.ingot, Materials.Aluminium, 1L), 12);
+        TFTreasure.hill3.rare.add(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 2L), 12);
+        TFTreasure.hill3.rare.add(GT_OreDictUnificator.get(OrePrefixes.ingot, Materials.Aluminiumoxide, 1L), 12);
```

Comment:
Aluminiumoxide Ingot doesn't sound right.

Reply:
It can be used as a material to contain high heat but it's very brittle and can be made only in a press, maybe can be used as a new material for coils? It should be impossible to bend and extrude it though (I'm not sure whether adding ingot adds all of those standard machine recipes automatically)

Reply:
Yes that is wrong need to change

Reply:
There is no ingot anyways

Reply:
changed back